models/booking.js

```diff
@@ -1,7 +1,20 @@
 const mongoose = require('mongoose');
 
 const bookingSchema = new mongoose.Schema({
-
+  author: {
+    id: {
+      type: mongoose.Schema.Types.ObjectId,
+      ref: 'user'
+    },
+    username: String,
+    required: true
+  },
+  startDate: {type: Date, required: true},
+  endDate: {type: Date, required: true},
+  price: {type: Number, required: true, min: 0.01},
+  origin: {type: String, required: true},
+  guests: {type: Number, required: true, min: 1},
+  nation: {type: String}
 });
 
 const booking = mongoose.model('Booking', bookingSchema);
```

Comment:
You can inline this to `module.exports`, no need for a new var. Even better, then you'll have `booking` var free so you can use it for schema so it's nice and short.